src/runway/runway.yml

```diff
@@ -0,0 +1,7 @@
+deployments:
+  - modules:
+      - path: sampleapp.cfn
+    regions:
+      - us-west-2
+    account-id:
+      dev: test
```

Comment:
Can we move this file to the tests fixtures directory or templates?

Reply:
looks like it was one of the files I was using during dev. my bad - i can remove it completely. it has no functional significance. was just something I was using to validate parsing.